tutorials/plot_background_filtering.py

```diff
@@ -894,6 +894,102 @@ def baseline_plot(x):
 #     (see also :func:`mne.filter.construct_iir_filter` for options).
 #     For more inforamtion, see e.g. `FieldTrip band-pass documentation`_.
 #
+# Reporting Filters
+# =================
+# On page 45 in Widmann *et al.* [7]_, there is a convenient list of important
+# filter parameters that should be reported with each publication:
+#
+#    1. filtertype (high-pass, low-pass, band-pass, band-stop, FIR, IIR)
+#    2. cutoff frequency (including definition)
+#    3. filter order (or length)
+#    4. roll-off or transition bandwidth
+#    5. passband ripple and stopband attenuation
+#    6. filter delay (zero-phase, linear-phase, non-linear phase) and causality
+#    7. direction of computation (one-pass forward/reverse,or two-pass forward and reverse)  # noqa
+#
+# In the following, we will address how to deal with these parameters in MNE:
```

Comment:
If we take @jona-sassenhagen's suggestion, then these can be shown in the log messages of a `create_filter(..., verbose=True)`

Reply:
Yes we can do it later